systems/plants/RigidBodyManipulator.cpp

```diff
@@ -273,6 +273,20 @@ RigidBodyManipulator::RigidBodyManipulator(int ndof, int num_featherstone_bodies
   resize(ndof,num_featherstone_bodies,num_rigid_body_objects,num_rigid_body_frames);
 }
 
+RigidBodyManipulator::RigidBodyManipulator(const std::string &urdf_filename)
+  :  collision_model(DrakeCollision::newModel()), collision_model_no_margins(DrakeCollision::newModel())
+{
+  num_dof=0; NB=0; num_bodies=0; num_frames=0;
+  a_grav << 0,0,0,0,0,-9.81;
```

Comment:
Is this default initialization going to bite us sometime in the future? Why is it special in that it doesn't have to be set by the URDF parser or constructModelmex?